docs/tutorials/installation.md

```diff
@@ -20,6 +31,14 @@ pip install --upgrade "synapseclient[pandas]"
 pip install --upgrade "synapseclient[pandas, pysftp, boto3]"
 ```
 
+> **NOTE** <br>
+> The `synapseclient` package may require loading shared libraries located in your system's `/usr/local/lib` directory. Some users working
+with `conda` have experienced issues with shared libraries not being found due to the system searching in the wrong locations. Although
+not recommended, one solution for this is manually configuring the `LD_LIBRARY_PATH` environment variable to point
+to the `/usr/local/lib` directory. [See here](https://github.com/conda/conda/issues/12800) for more context on this solution, and for alternatives.
```

Comment:
Because of Travis Hathaway's comment [here](https://github.com/conda/conda/issues/12800#issuecomment-1593231776) I did not feel comfortable adding a direct solution to this and opted for directing the user to this conversation found by @BryanFauble instead (thanks Bryan).